src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -831,7 +851,37 @@ private void validateLetterNotifications(List<String> expectedLetterTemplateIds,
                         "1",
                         "1",
                         "0"
-                },
+                }
+        };
+    }
+
+    @SuppressWarnings({"Indentation", "unused"})
+    private Object[] generateJointPartySubscriptionNotificationScenarios() {
+        return new Object[]{
+            new Object[]{
+                SUBSCRIPTION_UPDATED_NOTIFICATION,
```

Comment:
I'm confused what event we're testing, I thought it was jointPartySubscriptionUpdated not subscriptionUpdated